src/project/FileTreeView.js

```diff
@@ -109,6 +111,40 @@ define(function (require, exports, module) {
     }
 
     /**
+     * @private
+     *
+     * Create an appropriate div based thickness to indent the tree correctly.
+     *
+     * @param {int} depth - The depthness of the current node.
+     * @return {ReactComponent} - The resulting div.
+     */
+    function _createThickness(depth) {
+        return DOM.div({
+            className: "thickness",
```

Comment:
Rather than making it dependant on some css class, wouldn't it be better if we can define all the styles inside `style` property?